### DISCUSSION TITLE
[5.0] Mass Mail Users: 1 solution needed

### DISCUSSION THREAD

**Discussion by Kostelano** [joomla/joomla-cms#47086] [2. Cold Features]
### Steps to reproduce the issue
With the addition of email templates, there was some inconsistency or, more precisely, a collection of parameters not oriented in one place in the Mass Mail Users extension.

We go to the user settings on the appropriate tab, enter the prefix.

![Screenshot_2](https://user-images.githubusercontent.com/8440661/161788341-49155cfe-29a3-440e-b53e-333bbbeae15a.png)

Then we go to email templates - here we can also make adjustments to the subject or body of the message (essentially the same prefix / suffix).

![Screenshot_3](https://user-images.githubusercontent.com/8440661/161788411-30396528-bd74-4ca9-b1ff-83ddd8090eba.png)

![Screenshot_4](https://user-images.githubusercontent.com/8440661/161788484-6a538c00-8e6a-48a0-8794-8be194c8cabd.png)

![Screenshot_5](https://user-images.githubusercontent.com/8440661/161788605-ad2c6af1-9567-4bc1-b0d3-0398c93ee604.png)

As you can see, the real prefix (not 123 from the user parameters) that I added directly in the email templates (Something else) is missing from the email.

What is the point of "duplicating" parameters...?

I think that with the advent of email templates, we should remove the Mass Mail Users tab in the user settings, and leave only the `{SUBJECT}` and `{BODY}` tags in email templates.

**Comment by brianteeman** [comment]
**Mass Mail Options - prefix and suffix**
We can't really remove the ability to set these as it would be a disruptive backwards compatibility change
